test/extended/openstack/loadbalancers.go

```diff
@@ -382,6 +383,152 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb][Serial] The O
 			e2e.Logf("Pod %s successfully accessed through svc loadbalancer on ip %s and port %d", podName, fip.FloatingIP, svcPort)
 		})
 	}
+
+	// Only valid with the Octavia amphora provider. The OVN-Octavia provider does not
+	// support allowed_cidrs in the LB (https://issues.redhat.com/browse/OCPBUGS-2789)
+	g.It("should limit service access on an UDP Amphora LoadBalancer when an UDP LoadBalancer svc setting the loadBalancerSourceRanges spec is created on Openshift", func() {
```

Comment:
Why UDP? loadBalancerSourceRanges is independent from protocol, so I'd consider using TCP as it's just working better with Octavia in any version. ;)

Reply:
There is no such test for TCP yet so we prefer to test it even if it's with UDP. This service is not of type ETP:local so there should not be any issue as it won't rely on loadbalancer monitors.